src/extension.ts

```diff
@@ -48,14 +50,23 @@ export async function activate(context: vscode.ExtensionContext): Promise<Extens
     let signStatusBarObserver = new SignStatusBarObserver(signStatusBar, environmentController);
     eventStream.subscribe(signStatusBarObserver.eventHandler);
 
+    // Build component initialize
+    let diagnosticController = new DiagnosticController();
```

Comment:
>let [](start = 4, length = 3)

similar to JS, this could be `const`

Reply:
will add more tslint rule in the next PR

---
In reply to: [365068749](https://github.com/docascode/vscode-docs-build/pull/13#discussion_r365068749) [](ancestors = 365068749)